File: src/test/java/codility/lesson6/numberofdiskintersections/SolutionTest.java
package codility.lesson6.numberofdiskintersections;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class SolutionTest {

    @Test
    public void name() {
        int result = new Solution().solution(new int[]{1, 2147483647, 0});
        assertEquals(result, 2);
    }
}

File: src/test/java/pro/12919/SolutionTest.kt
package pro.`12919`


import org.junit.Assert.assertTrue
import org.junit.Test

class SolutionTest {
    @Test
    fun test1() {
        val solution = Solution()
        val answer = solution.solution(arrayOf("Jane","Kim"))
        assertTrue(answer.equals("김서방은 1에 있다"))
    }
}